.github/workflows/ci.yml

```diff
@@ -34,10 +34,8 @@ jobs:
       fail-fast: false
       matrix:
         arch: ["amd64", "arm64"]
-        stack-version: ["20", "22", "24"]
```

Comment:
I think we might want to wait until a bit later in May, since technically Ubuntu say May 2025 for Ubuntu 20.04:
https://ubuntu.com/about/release-cycle

...so there might be one or two more package updates this month (they don't say when in May though).

(18.04 and 20.04 are slight anomalies compared to LTS versions both before and after them, which instead say April as the month)

Reply:
For example there was a kernel update today (which isn't relevant for Docker images, since they don't include a kernel, but as an indication of updates still being released):
https://www.ubuntuupdates.org/package_logs?type=dists&vals=focal

Reply:
(All other Heroku-20 related cleanup is fine to do now though, such as buildpacks etc)

Reply:
Sounds good. Let's leave this comment thread open (block merging) and resolve it in June. I'm trying to pick off tasks from https://gus.lightning.force.com/lightning/r/ADM_Epic__c/a3QEE000000thsL2AQ/related/Work__r/view

Reply:
There was another base image update today, however, a few hours after there was yet another upstream Ubuntu 20.04 package release (I imagine one that just missed the end of the month so rolled over):

```
$ docker pull -q heroku/heroku:20 && docker run --rm heroku/heroku:20 bash -c 'apt-get update >/dev/null && apt list --upgradable'
...
Listing...
libvpx6/focal-security,focal-updates 1.8.2-1ubuntu0.4 amd64 [upgradable from: 1.8.2-1ubuntu0.3]
```

We might as well wait until after the 2025.06b release now.

Reply:
We're good to proceed with this now.